app_dart/lib/src/foundation/utils.dart

```diff
@@ -12,10 +12,17 @@ import 'package:github/github.dart';
 import 'package:googleapis/bigquery/v2.dart';
 import 'package:meta/meta.dart';
 import 'package:retry/retry.dart';
+import 'package:yaml/yaml.dart';
 
+import '../../protos.dart';
 import '../foundation/typedefs.dart';
+import '../request_handlers/flaky_handler_utils.dart';
```

Comment:
This is weird dependency, I will probably move the methods here

Reply:
Yeah, I know, but don't wanna to make this PR too complex.
Created https://github.com/flutter/flutter/issues/87287 to refactor.